modules/metering-troubleshooting.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-troubleshooting-debugging.adoc
+
+[id="metering-troubleshooting_{context}"]
+= Troubleshooting metering
+
+A common issue with metering is Pods failing to start. Pods may fail to start due to lack of resources, or if they have a dependency on a resource that does not exist, such as a `StorageClass` or `Secret`.
```

Comment:
@lamk, should we add a section for checking for problems associated with Secret`?  The document mentioned `StorageClass` or `Secret` issues can Pods failing to start.  It has `StorageClass` example but not `Secret` example .  WDYT?

Reply:
I think that's a good idea. I'll sync with you on the content because I'm not sure of the common errors or what people would see in the product.

Reply:
Honestly a lot of the StorageClass  troubleshooting stuff isn't necessary now that we can guarantee we're on OCP which ships with a default storageClass.